sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentAction.cs

```diff
@@ -51,7 +51,7 @@ public AnalyzeSentimentAction(AnalyzeSentimentOptions options)
         /// </para>
         /// </summary>
         /// <remarks>
-        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/> and up.
+        /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_2_Preview_2"/>, <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_04_01_Preview"/>, and up.
```

Comment:
Since the previews for `v3_2` aren't going to have a stable release, we'll want to remove mentions of them in favor of `V3_1` as we come across them as well.